client/src/components/profile/profile.styles.jsx

```diff
@@ -0,0 +1,27 @@
+import { makeStyles } from '@material-ui/styles';
+
+export const useStyles = makeStyles(theme => ({
+  unselected: {
+    backgroundColor: '#F5F8FA',
+    '&:hover': {
+      backgroundColor: '#fff',
+    },
+  },
+  selected: {
+    backgroundColor: '#fff',
+  },
+  rounded: {
+    borderRadius: 16,
+  },
+  padddd: {
```

Comment:
be careful with naming haha whats `padddd` and i noticed you named something else `padd` as well

Reply:
renamed to pad10 in d2b024c